packages/core/contracts/financial-templates/perpetual-multiparty/PerpetualInterface.sol

```diff
@@ -0,0 +1,16 @@
+pragma solidity ^0.6.0;
+pragma experimental ABIEncoderV2;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "../../common/implementation/FixedPoint.sol";
+
+/**
+ * @notice External methods that the FundingRateStore needs access to.
+ */
+interface PerpetualInterface {
+    function payFundingRateFees(FixedPoint.Unsigned memory amount) external;
```

Comment:
nit: should this method be called `retrieve` or `withdraw` rather than `pay` since the caller is pulling the funds?

Reply:
Yes I like `withdraw`